turbo.json

```
@@ -1,6 +1,6 @@
 {
```

```suggestion
    "build": {
      "dependsOn": ["^package", "^build"],
      "outputs": [".svelte-kit/**", ".vercel/**", "dist/**"],
      "env": ["HYGRAPH_TOKEN", "HYGRAPH_URL"]
    },
```

turbo also now requires `build.env` declaring platform environment variables like this

https://turbo.build/repo/docs/crafting-your-repository/using-environment-variables#platform-environment-variables